pkg/sql/logictest/tests/local/local_test: TestLogic_builtins failed

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21325368?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21325368?buildTab=artifacts#/) on master @ [52d26465d76704a09785a564d8d3a2ae28a8b22d](https://github.com/cockroachdb/cockroach/commits/52d26465d76704a09785a564d8d3a2ae28a8b22d):


```
                     ---
             2097152  31            0            0            0            0            1      1957888            1      1957888
                     ---
Bytes in small extent cache: 0
HPA shard stats:
  Purge passes: 0 (0 / sec)
  Purges: 0 (0 / sec)
  Hugeifies: 0 (0 / sec)
  Dehugifies: 0 (0 / sec)

  In full slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[13:15:28] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4858: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins2011564727
--- FAIL: TestLogic_builtins (14.06s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[13:15:24] SET TRACING = "on", cluster;;
[13:15:24] SELECT * FROM xy;;
[13:15:24] SET TRACING = "off";;
[13:15:24] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[13:15:24] 	-- FAIL
    logic.go:3264: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4337/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[13:15:24] 	-- FAIL
    logic.go:2420: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4337/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63234

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21327497?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21327497?buildTab=artifacts#/) on master @ [eb8f019262c438d83266cf3e1d668a00ff6399d8](https://github.com/cockroachdb/cockroach/commits/eb8f019262c438d83266cf3e1d668a00ff6399d8):


```
                     ---
             2097152  31            0            0            0            0            1      1978368            1      1978368
                     ---
Bytes in small extent cache: 0
HPA shard stats:
  Purge passes: 0 (0 / sec)
  Purges: 0 (0 / sec)
  Hugeifies: 0 (0 / sec)
  Dehugifies: 0 (0 / sec)

  In full slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[14:17:24] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4858: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1893003949
--- FAIL: TestLogic_builtins (15.04s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[14:17:20] SET TRACING = "on", cluster;;
[14:17:20] SELECT * FROM xy;;
[14:17:20] SET TRACING = "off";;
[14:17:20] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[14:17:20] 	-- FAIL
    logic.go:3264: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3831/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[14:17:20] 	-- FAIL
    logic.go:2420: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3831/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21327573?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21327573?buildTab=artifacts#/) on master @ [eb8f019262c438d83266cf3e1d668a00ff6399d8](https://github.com/cockroachdb/cockroach/commits/eb8f019262c438d83266cf3e1d668a00ff6399d8):


```
=== RUN   TestLogic_builtins
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1106583901
    test_log_scope.go:82: use -show-logs to present logs inline
[06:33:39] --- queries start here (file: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3707/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins)
[06:33:39] CREATE TABLE xy (x INT, y INT);
INSERT INTO xy VALUES (1, 2);;
[06:33:39] rewrote:
CREATE TABLE xy (x INT8, y INT8, FAMILY (y, x));
INSERT INTO xy VALUES (1, 2);


[06:33:42] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3707/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins with config local: 16 tests, 2 failures
[06:33:45] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4858: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1106583901
--- FAIL: TestLogic_builtins (10.68s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[06:33:41] SET TRACING = "on", cluster;;
[06:33:41] SELECT * FROM xy;;
[06:33:42] SET TRACING = "off";;
[06:33:42] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[06:33:42] 	-- FAIL
    logic.go:3264: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3707/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[06:33:42] 	-- FAIL
    logic.go:2420: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3707/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21329121?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21329121?buildTab=artifacts#/) on master @ [3399ae7c9d6e7b7c840a2355f9fbc50a4cf28ffe](https://github.com/cockroachdb/cockroach/commits/3399ae7c9d6e7b7c840a2355f9fbc50a4cf28ffe):


```
                     ---
             1048576  27            0            0            0            0            1      1044480            1      1044480
                     ---
Bytes in small extent cache: 0
HPA shard stats:
  Purge passes: 0 (0 / sec)
  Purges: 0 (0 / sec)
  Hugeifies: 0 (0 / sec)
  Dehugifies: 0 (0 / sec)

  In full slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[11:17:53] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4874: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins97354598
--- FAIL: TestLogic_builtins (15.03s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[11:17:49] SET TRACING = "on", cluster;;
[11:17:49] SELECT * FROM xy;;
[11:17:49] SET TRACING = "off";;
[11:17:49] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[11:17:49] 	-- FAIL
    logic.go:3280: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[11:17:49] 	-- FAIL
    logic.go:2432: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4092/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21330987?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21330987?buildTab=artifacts#/) on master @ [cb2979783e3bf83aaf3ee4de9c6c0f5fd968d56e](https://github.com/cockroachdb/cockroach/commits/cb2979783e3bf83aaf3ee4de9c6c0f5fd968d56e):


```
=== RUN   TestLogic_builtins
I260429 08:56:47.137803 1 (gostd) rand.go:288  [-] 1  random seed: 204566042648723522
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1353130822
    test_log_scope.go:82: use -show-logs to present logs inline
[08:56:51] --- queries start here (file: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3895/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins)
[08:56:51] CREATE TABLE xy (x INT, y INT);
INSERT INTO xy VALUES (1, 2);;
[08:56:51] rewrote:
CREATE TABLE xy (x INT8, y INT8, FAMILY (x, y));
INSERT INTO xy VALUES (1, 2);


[08:56:53] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3895/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins with config local: 16 tests, 2 failures
[08:56:56] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4874: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1353130822
--- FAIL: TestLogic_builtins (9.54s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[08:56:53] SET TRACING = "on", cluster;;
[08:56:53] SELECT * FROM xy;;
[08:56:53] SET TRACING = "off";;
[08:56:53] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[08:56:53] 	-- FAIL
    logic.go:3280: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3895/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[08:56:53] 	-- FAIL
    logic.go:2432: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3895/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21332731?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21332731?buildTab=artifacts#/) on master @ [3de22045c0bd228e9eddaa92510a01b14ff847bb](https://github.com/cockroachdb/cockroach/commits/3de22045c0bd228e9eddaa92510a01b14ff847bb):


```
=== RUN   TestLogic_builtins
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins471926488
    test_log_scope.go:82: use -show-logs to present logs inline
[11:25:59] setting distsql_workmem='26009B';
[11:25:59] --- queries start here (file: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4250/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins)
[11:25:59] CREATE TABLE xy (x INT, y INT);
INSERT INTO xy VALUES (1, 2);;
[11:25:59] rewrote:
CREATE TABLE xy (x INT8, y INT8, FAMILY (x, y));
INSERT INTO xy VALUES (1, 2);


[11:26:01] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4250/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins with config local: 16 tests, 2 failures
[11:26:03] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:4874: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins471926488
--- FAIL: TestLogic_builtins (7.09s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[11:26:01] SET TRACING = "on", cluster;;
[11:26:01] SELECT * FROM xy;;
[11:26:01] SET TRACING = "off";;
[11:26:01] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[11:26:01] 	-- FAIL
    logic.go:3280: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4250/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[11:26:01] 	-- FAIL
    logic.go:2432: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4250/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21334140?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21334140?buildTab=artifacts#/) on master @ [7cca024537d0efe15f607f4da37f401d198763ff](https://github.com/cockroachdb/cockroach/commits/7cca024537d0efe15f607f4da37f401d198763ff):


```
=== RUN   TestLogic_builtins
I260501 07:17:39.379252 1 (gostd) rand.go:288  [-] 1  random seed: 5711138537925987640
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins3736480290
    test_log_scope.go:82: use -show-logs to present logs inline
[07:17:43] setting distsql_workmem='10651B';
[07:17:43] --- queries start here (file: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3823/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins)
[07:17:43] CREATE TABLE xy (x INT, y INT);
INSERT INTO xy VALUES (1, 2);;
[07:17:43] rewrote:
CREATE TABLE xy (x INT8, y INT8, FAMILY (x, y));
INSERT INTO xy VALUES (1, 2);


[07:17:45] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3823/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins with config local: 16 tests, 2 failures
[07:17:48] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins3736480290
--- FAIL: TestLogic_builtins (9.14s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[07:17:45] SET TRACING = "on", cluster;;
[07:17:45] SELECT * FROM xy;;
[07:17:45] SET TRACING = "off";;
[07:17:45] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[07:17:45] 	-- FAIL
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3823/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[07:17:45] 	-- FAIL
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3823/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21335150?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21335150?buildTab=artifacts#/) on master @ [5b0bc8ba6eb54d9e5080371c6dab17d465391d08](https://github.com/cockroachdb/cockroach/commits/5b0bc8ba6eb54d9e5080371c6dab17d465391d08):


```
                     ---
              524288  23            0            0            0            0            1       516096            1       516096
                     ---
Bytes in small extent cache: 0
HPA shard stats:
  Purge passes: 0 (0 / sec)
  Purges: 0 (0 / sec)
  Hugeifies: 0 (0 / sec)
  Dehugifies: 0 (0 / sec)

  In full slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[06:24:47] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins142409713
--- FAIL: TestLogic_builtins (14.49s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[06:24:42] SET TRACING = "on", cluster;;
[06:24:42] SELECT * FROM xy;;
[06:24:43] SET TRACING = "off";;
[06:24:43] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[06:24:43] 	-- FAIL
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3729/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[06:24:43] 	-- FAIL
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3729/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21335601?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21335601?buildTab=artifacts#/) on master @ [9709f69a4fd6b6873937d9b3e9e27c825bb215d7](https://github.com/cockroachdb/cockroach/commits/9709f69a4fd6b6873937d9b3e9e27c825bb215d7):


```
=== RUN   TestLogic_builtins
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins3137225532
    test_log_scope.go:82: use -show-logs to present logs inline
[06:17:29] --- queries start here (file: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3687/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins)
[06:17:29] CREATE TABLE xy (x INT, y INT);
INSERT INTO xy VALUES (1, 2);;
[06:17:29] rewrote:
CREATE TABLE xy (x INT8, y INT8, FAMILY (x), FAMILY (y));
INSERT INTO xy VALUES (1, 2);


[06:17:32] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3687/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins with config local: 16 tests, 2 failures
[06:17:35] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins3137225532
--- FAIL: TestLogic_builtins (11.66s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[06:17:32] SET TRACING = "on", cluster;;
[06:17:32] SELECT * FROM xy;;
[06:17:32] SET TRACING = "off";;
[06:17:32] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[06:17:32] 	-- FAIL
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3687/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[06:17:32] 	-- FAIL
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3687/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local/local_test.TestLogic_builtins [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21336252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21336252?buildTab=artifacts#/) on master @ [be07a1c881a988095f0b16ea3a5b7cb54a341bc9](https://github.com/cockroachdb/cockroach/commits/be07a1c881a988095f0b16ea3a5b7cb54a341bc9):


```
                     ---
             2097152  31            0            0            0            0            1      1978368            1      1978368
                     ---
Bytes in small extent cache: 0
HPA shard stats:
  Purge passes: 0 (0 / sec)
  Purges: 0 (0 / sec)
  Hugeifies: 0 (0 / sec)
  Dehugifies: 0 (0 / sec)

  In full slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[16:39:04] --- total progress: 16 statements
--- total: 16 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/8dd27277675b679e2ab60da70e56a07e/logTestLogic_builtins1888343906
--- FAIL: TestLogic_builtins (15.27s)
=== RUN   TestLogic_builtins/clear_query_plan_cache
[16:39:00] SET TRACING = "on", cluster;;
[16:39:00] SELECT * FROM xy;;
[16:39:00] SET TRACING = "off";;
[16:39:00] SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';;
[16:39:00] 	-- FAIL
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4164/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:70: SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
        expected:
            query cache miss
            query cache add
            
        but found (query options: "") :
            query cache miss: SELECT * FROM xy
            query cache add: SELECT * FROM xy
            
[16:39:00] 	-- FAIL
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4164/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/builtins:76: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_builtins/clear_query_plan_cache (0.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_builtins.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/builtins
+++ b/pkg/sql/logictest/testdata/logic_test/builtins
@@ -66,8 +66,9 @@ statement ok
 SET TRACING = "off";
 
 # The first execution should miss the query cache.
+# Strip suffix after "query cache ..." because nightly tests with high vmodule setting appends the SQL stmt.
 query T
-SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
+SELECT regexp_replace(message, ':.*', '') FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
 ----
 query cache miss
 query cache add
@@ -83,7 +84,7 @@ SET TRACING = "off";
 
 # The second execution should hit the query cache.
 query T
-SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
+SELECT regexp_replace(message, ':.*', '') FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
 ----
 query cache hit
 
@@ -101,7 +102,7 @@ SET TRACING = "off";
 
 # After running the builtin, the query cache should be cleared.
 query T
-SELECT message FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
+SELECT regexp_replace(message, ':.*', '') FROM [SHOW TRACE FOR SESSION] WHERE message LIKE '%query cache%';
 ----
 query cache miss
 query cache add

